.gitignore

```diff
@@ -2,4 +2,5 @@
 /composer.lock
 /clover.xml
 /tests/Zend/Filter/_files/Users
+/tests/Zend/Filter/_files/Volumes
```

Comment:
This, and the one above it, are due to temp files being created during the test run.

Should look into changing the location of that file in the tests, or at least have it clean it up...